pg/migrate/010-proxy-proxy-info.sql

```diff
@@ -0,0 +1,11 @@
+CREATE SCHEMA proxy;
+
+CREATE TABLE proxy.proxy_info (
+  address      character varying(66),
+  tag          character varying(40),
+  tx           character varying(66),
+  from_address character varying(66)
+);
+
+CREATE INDEX proxy_proxy_info_address_index ON proxy.proxy_info(address);
```

Comment:
are there any other columns that we want to use in queries? it. do we need more indexes?

Reply:
what about tx? it's used in join later